### PR TITLE
Homer-541/DITAP page styling fixs

### PR DESCRIPTION
**Pull request by jaube05**
Updating offering template and styles for offering page for desktop, tablet and mobile

### CODE REVIEW

File: src/sass/pages/offering/_value-props.scss [hunk 1]
```diff
@@ -68,6 +68,7 @@
         text-align: left;
         font-size: 16px;
         font-weight: 300;
+        padding-top: 8px;
```
Comment (brdunfield) [inline src/sass/pages/offering/_value-props.scss:71]: I'm not sure why, but in Google Chrome I see that for some reason there's already a 6px gap between the image and the caption.
I'm not sure whether the desired 8px in the ticket is accounting for this (and so therefore the 14px we have in this PR is acceptable), or whether we need to compensate for that and just add 2 px?

File: src/sass/pages/offering/_value-props.scss [hunk 2]
```diff
@@ -12,8 +12,8 @@
     padding-top: 200px;
   }
   @media(min-width: 1200px) {
-    padding-top: 192px;
-    padding-bottom: 100px;
+    padding-top: 164px;
+    padding-bottom: 64px;
```
Comment (brdunfield) [inline src/sass/pages/offering/_value-props.scss:16]: Again, another weird artifact I'm seeing here is that the bottom padding seems to overlap the next section, meaning the whitespace here is likely less than 64px.

Screen Shot 2022-12-06 at 12 49 13 PM

**Comment by brdunfield** [comment]
> "See FAQs" should not be cut off on the right side

I'm still observing this in the mobile view. It just looks like the buttons / page are a pixel or two wider than the screen in mobile view (regardless of specific resolution).

**Comment by brdunfield** [comment]
> Whitespace between the bottom of the white alumni box and "Frequenty asked questions" should be 40px

It looks like there's a 20px margin and a 28px margin on the desktop view, which adds up to 48px.

File: src/sass/pages/offering/_value-props.scss (hunk 1 above)
Comment (jaube05) [reply src/sass/pages/offering/_value-props.scss]: Fixed this to be 2px

File: src/sass/pages/offering/_value-props.scss (hunk 2 above)
Comment (jaube05) [reply src/sass/pages/offering/_value-props.scss:16]: Adjusted the container below with tabs so padding is correct.

**Comment by jaube05** [comment]
@brdunfield 

> See FAQs" should not be cut off on the right side

Fixed.

> Whitespace between the bottom of the white alumni box and "Frequenty asked questions" should be 40px

Im only seeing 40px margin above the FAQ title, I posted a pic in the [Jira ticket ](https://civicactions-home.atlassian.net/jira/software/c/projects/HOMER20/boards/46?modal=detail&selectedIssue=HOMER20-541)

Also having issue with Strapi, so firebase is not working.

**Comment by brdunfield** [comment]
For the instructor bios, does "it should not be scrollable" mean the page should not be allowed to scroll while the modal is open? Everything else on this point looks good!

**Comment by brdunfield** [comment]
> @brdunfield
> 
> > See FAQs" should not be cut off on the right side
> 
> Fixed.
> 
> > Whitespace between the bottom of the white alumni box and "Frequenty asked questions" should be 40px
> 
> Im only seeing 40px margin above the FAQ title, I posted a pic in the [Jira ticket ](https://civicactions-home.atlassian.net/jira/software/c/projects/HOMER20/boards/46?modal=detail&selectedIssue=HOMER20-541)
> 
> Also having issue with Strapi, so firebase is not working.

I see the 40px now on mobile as well!

**Comment by jaube05** [comment]
@brdunfield 

> For the instructor bios, does "it should not be scrollable" mean the page should not be allowed to scroll while the modal is open? Everything else on this point looks good!

The bios body is very short to they want the bios in the modal to take the height of the info with no big white space making you scroll to see whole screen. Thats my assumption. 
Thank for you CR :)

**Pull request merged**